File: VB/WindowsFormsApplication1/CustomObjects.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace WindowsFormsApplication1
    #Region "#customappointment"
    Public Class CustomAppointment
        Private m_Start As Date
        Private m_End As Date
        Private m_Subject As String
        Private m_Status As Integer
        Private m_Description As String
        Private m_Label As Integer
        Private m_Location As String
        Private m_Allday As Boolean
        Private m_EventType As Integer
        Private m_RecurrenceInfo As String
        Private m_ReminderInfo As String
        Private m_OwnerId As String


        Public Property StartTime() As Date
            Get
                Return m_Start
            End Get
            Set(ByVal value As Date)
                m_Start = value
            End Set
        End Property
        Public Property EndTime() As Date
            Get
                Return m_End
            End Get
            Set(ByVal value As Date)
                m_End = value
            End Set
        End Property
        Public Property Subject() As String
            Get
                Return m_Subject
            End Get
            Set(ByVal value As String)
                m_Subject = value
            End Set
        End Property
        Public Property Status() As Integer
            Get
                Return m_Status
            End Get
            Set(ByVal value As Integer)
                m_Status = value
            End Set
        End Property
        Public Property Description() As String
            Get
                Return m_Description
            End Get
            Set(ByVal value As String)
                m_Description = value
            End Set
        End Property
        Public Property Label() As Integer
            Get
                Return m_Label
            End Get
            Set(ByVal value As Integer)
                m_Label = value
            End Set
        End Property
        Public Property Location() As String
            Get
                Return m_Location
            End Get
            Set(ByVal value As String)
                m_Location = value
            End Set
        End Property
        Public Property AllDay() As Boolean
            Get
                Return m_Allday
            End Get
            Set(ByVal value As Boolean)
                m_Allday = value
            End Set
        End Property
        Public Property EventType() As Integer
            Get
                Return m_EventType
            End Get
            Set(ByVal value As Integer)
                m_EventType = value
            End Set
        End Property
        Public Property RecurrenceInfo() As String
            Get
                Return m_RecurrenceInfo
            End Get
            Set(ByVal value As String)
                m_RecurrenceInfo = value
            End Set
        End Property
        Public Property ReminderInfo() As String
            Get
                Return m_ReminderInfo
            End Get
            Set(ByVal value As String)
                m_ReminderInfo = value
            End Set
        End Property
        Public Property OwnerId() As String
            Get
                Return m_OwnerId
            End Get
            Set(ByVal value As String)
                m_OwnerId = value
            End Set
        End Property

        Public Sub New()
        End Sub
    End Class
    #End Region '  #customappointment

    #Region "#customresource"
    Public Class CustomResource
        Private m_name As String
        Private m_res_id As Integer

        Public Property Name() As String
            Get
                Return m_name
            End Get
            Set(ByVal value As String)
                m_name = value
            End Set
        End Property
        Public Property ResID() As Integer
            Get
                Return m_res_id
            End Get
            Set(ByVal value As Integer)
                m_res_id = value
            End Set
        End Property

        Public Sub New()
        End Sub
    End Class
    #End Region ' #customresource
End Namespace

File: CS/WindowsFormsApplication1/CustomObjects.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WindowsFormsApplication1 {
    #region #customappointment
    public class CustomAppointment {
        private DateTime m_Start;
        private DateTime m_End;
        private string m_Subject;
        private int m_Status;
        private string m_Description;
        private int m_Label;
        private string m_Location;
        private bool m_Allday;
        private int m_EventType;
        private string m_RecurrenceInfo;
        private string m_ReminderInfo;
        private string m_OwnerId;


        public DateTime StartTime { get { return m_Start; } set { m_Start = value; } }
        public DateTime EndTime { get { return m_End; } set { m_End = value; } }
        public string Subject { get { return m_Subject; } set { m_Subject = value; } }
        public int Status { get { return m_Status; } set { m_Status = value; } }
        public string Description { get { return m_Description; } set { m_Description = value; } }
        public int Label { get { return m_Label; } set { m_Label = value; } }
        public string Location { get { return m_Location; } set { m_Location = value; } }
        public bool AllDay { get { return m_Allday; } set { m_Allday = value; } }
        public int EventType { get { return m_EventType; } set { m_EventType = value; } }
        public string RecurrenceInfo { get { return m_RecurrenceInfo; } set { m_RecurrenceInfo = value; } }
        public string ReminderInfo { get { return m_ReminderInfo; } set { m_ReminderInfo = value; } }
        public string OwnerId { get { return m_OwnerId; } set { m_OwnerId = value; } }

        public CustomAppointment() {
        }
    }
    #endregion  #customappointment

    #region #customresource
    public class CustomResource {
        private string m_name;
        private int m_res_id;
        
        public string Name { get { return m_name; } set { m_name = value; } }
        public int ResID { get { return m_res_id; } set { m_res_id = value; } }

        public CustomResource() {
        }
    }
    #endregion #customresource
}
